### PR TITLE
Remove sudo from Travis

### PR DESCRIPTION
**Pull request by bobisjan**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +26,7 @@ branches:
     - /^v\d+\.\d+\.\d+/
 
 jobs:
-  fail_fast: true
+  fast_finish: true
```
Comment (bobisjan) [inline .travis.yml:29]: https://docs.travis-ci.com/user/customizing-the-build#fast-finishing

File: .travis.yml [hunk 2]
```diff
@@ -5,7 +5,6 @@ node_js:
   # so that your addon works for all apps
   - "8"
 
-sudo: true
```
Comment (bobisjan) [inline .travis.yml:8]: https://changelog.travis-ci.com/linux-builds-run-on-vms-by-default-77106

**Comment by bobisjan** [comment]
The build is green https://travis-ci.org/github/willviles/ember-useragent/builds/687459676.

File: .travis.yml (hunk 2 above)
Comment (willviles) [reply .travis.yml:8]: Aha! Excellent, thanks.

**Pull request merged**